src/libNode/MicroBlockPostProcessing.cpp

```diff
@@ -280,11 +291,24 @@ bool Node::ProcessMicroBlockConsensusCore(const bytes& message,
 
     {
       lock_guard<mutex> g(m_mutexShardMember);
+      // To DS -> ProcessMicroBlock
       DataSender::GetInstance().SendDataToOthers(
           *m_microblock, *m_myShardMembers, ds_shards, t_blocks,
           m_mediator.m_lookup->GetLookupNodes(),
           m_mediator.m_txBlockChain.GetLastBlock().GetBlockHash(),
           m_consensusMyID, composeMicroBlockMessageForSender, false, nullptr);
+      // To Lookup -> ProcessMBnForwardTxn
+      DataSender::GetInstance().SendDataToOthers(
+          *m_microblock, *m_myShardMembers, {}, {},
+          m_mediator.m_lookup->GetLookupNodes(),
+          m_mediator.m_txBlockChain.GetLastBlock().GetBlockHash(),
+          m_consensusMyID, composeMBnForwardTxnMessageForSender, false,
+          SendDataToLookupFuncDefault, sendMbnFowardTxnToShardNodes);
+      // pending Txns
+      if (!IsUnconfirmedTxnEmpty()) {
+        SendPendingTxnToLookup();
```

Comment:
Just to check: We still have code for `SendPendingTxnToLookup` inside `Node::ProcessFinalBlockCore`. I'm guessing that code will now be relevant only for DS committee?

Reply:
Yes we still need to check this on shard, in case the microblock from this shard is not included in the FinalBlock.